releases/patch-releases.md

```diff
@@ -59,47 +59,57 @@ releases may also occur in between these.
 
 ### 1.19
 
-Next patch release is **1.19.1**.
+Next patch release is **1.19.3**
 
-| Patch Release | Cherry-picks deadline | Target date |
-| --- | --- | --- |
-| 1.19.2 | 2020-09-11 | 2020-09-16 |
-| 1.19.1 | 2020-09-04 | 2020-09-09 |
+End of Life for **1.19** is **2021-09-30**
+
+| PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
+|--- |--- |--- |
+| 1.19.3        | 2020-10-09           | 2020-10-14  |
+| 1.19.2        | 2020-09-11           | 2020-09-16  |
+| 1.19.1        | 2020-09-04           | 2020-09-09  |
 
 ### 1.18
 
-Next patch release is **1.18.9**.
+Next patch release is **1.18.10**
 
-| Patch Release | Cherry-picks deadline | Target date |
-| --- | --- | --- |
-| 1.18.9 | 2020-09-11 | 2020-09-16 |
-| 1.18.8 | N/A | 2020-08-13 |
-| 1.18.7 | 2020-08-07 | 2020-08-12 |
-| 1.18.6 | 2020-07-10 | 2020-07-15 |
-| 1.18.5 | 2020-06-25 | 2020-06-26 |
-| 1.18.4 | 2020-06-12 | 2020-06-17 |
-| 1.18.3 | 2020-05-15 | 2020-05-20 |
-| 1.18.2 | 2020-04-13 | 2020-04-16 |
-| 1.18.1 | 2020-04-06 | 2020-04-08 |
+End of Life for **1.18** is **TBD**
+
+| PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
+|--- |--- |--- |
+| 1.18.10       | 2020-10-09           | 2020-10-14  |
+| 1.18.9        | 2020-09-11           | 2020-09-16  |
+| 1.18.8        | N/A                  | 2020-08-13  |
+| 1.18.7        | 2020-08-07           | 2020-08-12  |
+| 1.18.6        | 2020-07-10           | 2020-07-15  |
+| 1.18.5        | 2020-06-25           | 2020-06-26  |
+| 1.18.4        | 2020-06-12           | 2020-06-17  |
+| 1.18.3        | 2020-05-15           | 2020-05-20  |
+| 1.18.2        | 2020-04-13           | 2020-04-16  |
+| 1.18.1        | 2020-04-06           | 2020-04-08  |
 
 ### 1.17
 
-Next patch release is **1.17.12**.
+Next patch release is **1.17.13**
+
+End of Life for **1.17** is **TBD**
+
+| PATCH RELEASE | CHERRY PICK DEADLINE | TARGET DATE |
+|--- |--- |--- |
+| 1.17.13       | 2020-10-09 | 2020-10-14  |
+| 1.17.12       | 2020-09-11 | 2020-09-16  |
+| 1.17.11       | N/A | 2020-08-13  |
+| 1.17.10       | 2020-08-07 | 2020-08-12  |
+| 1.17.9        | 2020-07-10 | 2020-07-15  |
+| 1.17.8        | 2020-06-25 | 2020-06-26  |
+| 1.17.7        | 2020-06-12 | 2020-06-17  |
+| 1.17.6        | 2020-05-15 | 2020-05-20  |
+| 1.17.5        | 2020-04-13 | 2020-04-16  |
+| 1.17.4        | 2020-03-09 | 2020-03-12  |
+| 1.17.3        | 2020-02-07 | 2020-02-11  |
+| 1.17.2        | [No-op release](https://groups.google.com/d/topic/kubernetes-dev/Mhpx-loSBns/discussion) | 2020-01-21  |
```

Comment:
We may need to not allow variable length table cell content as this will lead to tooling adding/removing spaces in all the other cells complicating diffs and reviews.  A simple "N/A" should suffice, imho.  There are ways to get at what happened if somebody is curious.